sdk/core/Azure.Core.Expressions.DataFactory/src/DataFactoryKeyVaultSecretReference.Serialization.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Azure.Core.Expressions.DataFactory
+{
+    [JsonConverter(typeof(DataFactoryKeyVaultSecretReferenceConverter))]
```

Comment:
JsonConverter was added to allow for public serialization from the Azure.ResouceManager.DataFactory library.